lib/index.js

```diff
@@ -49,6 +49,11 @@ class Ream extends Event {
     this.options = merge(
       {
         entry: 'index.js',
+        fsRoutes: {
+          baseDir: 'pages',
+          basePath: '/',
```

Comment:
is this the same thing as `router.options.base`? maybe we can get it from the `router` instead.

Reply:
I don't think so; `router.options.base` is runtime and this `basePath` is compile-time (it's imprinted in `.ream/routes.js`). Unless we go with some routes post-processing in `inject.js` it's not possible to achieve.

Also, I don't think it's really needed. The defaults are good for 95% use cases; for the rest additional flexibility wouldn't hurt.

Reply:
Moreover, from what I believe `router.options.base` is automatically applied in runtime to all routes. So it's not the same thing. For example, I may set `router.options.base` to `/foo` and `fsRoutes.baseDir` to `/bar` and then `pages/baz.vue` will open at `/foo/bar/baz`.